Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/TailCalls.dfy.expect

```diff
@@ -28,4 +28,5 @@ TailCalls.dfy(462,15): Error: a recursive call in this context is not recognized
 TailCalls.dfy(110,14): Error: the recursive call to '_ctor' is not tail recursive, because the assignment of the LHS happens after the call
 TailCalls.dfy(126,19): Error: the recursive call to 'Compute' is not tail recursive because the actual type argument is not the formal type parameter 'G'
 TailCalls.dfy(139,17): Error: the recursive call to 'Run' is not tail recursive because the actual type argument 1 is not the formal type parameter 'G'
-30 resolution/type errors detected in TailCalls.dfy
+TailCalls.dfy(171,13): Error: cannot use naked function in recursive setting. Possible solution: eta expansion.
```

Comment:
Can we avoid the use of the word naked here? Can I find more information on what the problem is and the  possible solution? Maybe add a reference to documentation?

Reply:
I left this as is. This PR does not introduce the phrase "naked function". It would be good to revisit, and indeed to say something about it in the documentation.